forge/db/controllers/Team.js

```diff
@@ -90,8 +90,9 @@ module.exports = {
             }
             await userRole.destroy()
 
+            await app.db.controllers.StorageSession.removeUserFromTeamSessions(user, team)
```

Comment:
Wouldn't the API: `await team.removeUserFromSessions(user)` be nicer, and make the model fat and keep the controller skinny?

https://medium.com/marmolabs/skinny-models-skinny-controllers-fat-services-e04cfe2d6ae

Reply:
We already have similar code in the controller, this keeps it all in the same place